src/jobs/workflow-collector.yml

```diff
@@ -3,20 +3,20 @@ description: >
 parameters:
   workflow-collector:
     type: env_var_name
-    default: "WORKFLOW_HTTP_SOURCE"
-    description: "Enter the HTTP Source endpoint generated by the Sumologic setup wizard for Workflows."
+    default: 'WORKFLOW_HTTP_SOURCE'
```

Comment:
Let's revert the change from double to single quotes to clean up this PR and stay in line with the rest of the orb.

Reply:
done